aten/src/ATen/native/cuda/DilatedMaxPool2d.cu

```diff
@@ -333,7 +329,7 @@ const Tensor& indices) {
   const int64_t outputHeight = output.size(-2);
   const int64_t outputWidth = output.size(-1);
 
-  Tensor input = input_.contiguous(memory_format);
+  const Tensor input = input_.contiguous(memory_format);
```

Comment:
nit: the const here doesn't do anything. (I know there are some preexisting occurrences of this, they are wrong)